includes/media_bynder.formatters.inc

```diff
@@ -74,3 +65,109 @@ function media_bynder_file_formatter_image_settings($form, &$form_state, $settin
     );
     return $element;
 }
+
+function media_bynder_field_info_alter(&$info) {
+    //TODO look up widget name
+   $info['image']['default widget'] = 'media_bynder';
+}
+
+/**
+ * Implements hook_image_default_styles().
+ * Add an example image style for drupal-media-bynder
+ */
+
+function media_bynder_image_default_styles() {
+    variable_set('image_style_preview_image', drupal_get_path('module', 'image') . '/sample.png');
+    $styles = array();
+
+    $styles['bynder_drupal_example'] = array(
+        'label' => t('Bynder Example (400x400)'),
+        'effects' => array(
+            array(
+                'name' => 'bynder_custom_resolution',
+                'data' => array(
+                    'width' => 400,
+                    'height' => 400
+                ),
+                'weight' => 0,
+            ),
+        ),
+    );
+    return $styles;
+}
+
+/**
+ * Implements hook_image_effect_info().
+ * declare effect callbacks
+ */
+function media_bynder_image_effect_info() {
+    return array(
+        'bynder_custom_resolution' => array(
+            'label' => t('Bynder custom resolution'), 
+            'help' => t('Creates a call to the public Bynder function in /additional/, make sure this is properly configured or ask the Bynder support team.<br>Please note that in order for this function to work you need to call this effect only as it uses an external image from Bynder. Due to this external nature additional drupal effects can\'t be applied'),
+            'effect callback'=> 'media_bynder_custom_resolution_effect',
+            'dimensions callback' => 'media_bynder_custom_resolution_dimensions',
+            'form callback'=> 'image_resize_form',
+        )
+    );
+}
+
+/**
+ * Implements callback 'effect callback'
+ * will return the image from the API, instead of modifying the image as intended
+ */
+function media_bynder_custom_resolution_effect(&$image, $data) {
+    if (!media_bynder_custom_resolution($image)) {
+        watchdog('image', 'he image %file could not be colorize because the imagefilter() function is not available in this PHP installation.', array('%file' => $image->source), WATCHDOG_ERROR);
```

Comment:
*T*he image %file could not be colorize*d* because the imagefilter() function is not available in this PHP installation.